### PR TITLE
RF_06.005.03 | Delete Organization Folder from project status table o…

### PR DESCRIPTION
**Pull request by Alzhan-B**
[RF_06.005.03 | Delete Organization Folder from project status table on Dashboard page](https://github.com/RedRoverSchool/JenkinsQA_JS_2024_fall/issues/473)

### CODE REVIEW

**Review by AnDerova — APPROVED** [review]

File: cypress/pageObjects/DashboardPage.js [hunk 1]
```diff
@@ -19,10 +19,12 @@ class DashboardPage {
   getAllJobNames = () => cy.get('.jenkins-table__link span')
   getLogOutButton = () => cy.get('a[href="/logout"]')
   getDeleteProjectDropdownMenuItem = () => cy.get('button.jenkins-dropdown__item ').contains('Delete Project');
+  getDeleteOrganizationFolderDropdownMenuItem = () => cy.get('[class="jenkins-dropdown__item "]').contains('Delete Organization Folder');
   getCancelProjectDeletingButton = () => cy.get('button[data-id="cancel"]');
   getSubmitProjectDeletingButton = () => cy.get('button[data-id="ok"]');
   getWelcomeToJenkins = () => cy.get('.empty-state-block h1');
-
+  getJenkinsHomeLink = () => cy.get('[id="jenkins-home-link"]')
```
Comment (ivanlashkevich) [inline cypress/pageObjects/DashboardPage.js:26]: We already have this locator named 'getJenkinsLogo'. As the logo relates to the header it is located in the Header.js file. So, delete the new added locator 'getJenkinsHomeLink' and replace in your test with the existing one.

File: cypress/pageObjects/DashboardPage.js [hunk 2]
```diff
@@ -93,9 +95,20 @@ class DashboardPage {
   }
 
   clickSubmitDeletingButton() {
-    this.getSubmitProjectDeletingButton().click()
-    return this
+    this.getSubmitProjectDeletingButton().click();
+    return this;
+  }
+
+  clickJenkinsHomeLinkImage() {
```
Comment (ivanlashkevich) [inline cypress/pageObjects/DashboardPage.js:102]: We also have such method 'clickJenkinsLogo ()' that redirects the user to the Dashboard page. The same as the locator, it is located in the 'Header.js' file. So, replace it in your test.

**Review by ivanlashkevich — CHANGES_REQUESTED** [review]
And could you please:
- fill in the description field in your github board card;
- write down your 'RF' to your US card.

File: cypress/pageObjects/DashboardPage.js (hunk 1 above)
Comment (Alzhan-B) [reply cypress/pageObjects/DashboardPage.js:26]: Thanks, Ivan, for pointing this out. Good catch! I deleted my selector from DashboardPage.js and used the one from the Header.js.

File: cypress/pageObjects/DashboardPage.js (hunk 2 above)
Comment (Alzhan-B) [reply cypress/pageObjects/DashboardPage.js:102]: Thanks! I also deleted my "clickJenkinsHomeLinkImage()" method from Dashboard.js and used the one you recommended from the Header.js -  'clickJenkinsLogo ()'.

**Comment by Alzhan-B** [comment]
> And could you please:
> 
> * fill in the description field in your github board card;
> * write down your 'RF' to your US card.

Could you elaborate please? My RF card has links to 1) PR, 2) US and 3) TC that is being refactored. I am not sure I am following you. Thanks!

File: cypress/pageObjects/DashboardPage.js (hunk 1 above)
Comment (ivanlashkevich) [reply cypress/pageObjects/DashboardPage.js]: Alzhan, good job! Thank you! Approved.

**Review by ivanlashkevich — APPROVED** [review]

**Comment by ivanlashkevich** [comment]
> > And could you please:
> > 
> > * fill in the description field in your github board card;
> > * write down your 'RF' to your US card.
> 
> Could you elaborate please? My current RF_06.005.03 card has links to 1) PR, 2) US, and 3) TC that is being refactored. I am not sure I am following you. What needs to be corrected? Thanks!

After the 'RF' github board card has been created, you shoul go to the US card, scroll down to 'Add a comment' section and write down your 'RF number and test case name' (e.g. RF_06.005.03 Delete Organization Folder from project status table on Dashboard page)

**Comment by Alzhan-B** [comment]
> > > And could you please:
> > > 
> > > * fill in the description field in your github board card;
> > > * write down your 'RF' to your US card.
> > 
> > 
> > Could you elaborate please? My current RF_06.005.03 card has links to 1) PR, 2) US, and 3) TC that is being refactored. I am not sure I am following you. What needs to be corrected? Thanks!
> 
> After the 'RF' github board card has been created, you should go to the US card, scroll down to 'Add a comment' section and write down your 'RF number and test case name' (e.g. RF_06.005.03 Delete Organization Folder from project status table on Dashboard page). [Follow the link.](https://github.com/RedRoverSchool/JenkinsQA_JS_2024_fall/issues/34)

Thank you, Ivan! Done.

File: cypress/pageObjects/DashboardPage.js [hunk 3]
```diff
@@ -21,11 +21,12 @@ class DashboardPage {
   getAllJobNames = () => cy.get('.jenkins-table__link span')
   getLogOutButton = () => cy.get('a[href="/logout"]')
   getDeleteProjectDropdownMenuItem = () => cy.get('button.jenkins-dropdown__item ').contains('Delete Project');
+  getDeleteOrganizationFolderDropdownMenuItem = () => cy.get('[class="jenkins-dropdown__item "]').contains('Delete Organization Folder');
   getCancelProjectDeletingButton = () => cy.get('button[data-id="cancel"]');
   getSubmitProjectDeletingButton = () => cy.get('button[data-id="ok"]');
   getWelcomeToJenkinsHeadline = () => cy.get('.empty-state-block h1');
-
-
+  getWelcomeToJenkins = () => cy.get('.empty-state-block h1');
```
Comment (Negrescu29) [inline cypress/pageObjects/DashboardPage.js:28]: @Alzhan-B 

you introduced a duplicate locator `getWelcomeToJenkins` an identical locator is just one line above `getWelcomeToJenkinsHeadline` when you a chance would you please remove the duplicated locator? thanks

**Pull request merged**